2020/07/17 07:05:52 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET /auth HTTP/1.1", upstream: "http://127.0.0.1:3010/auth", host: "localhost:8080"
2020/07/17 07:07:20 [error] 27#27: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET /auth HTTP/1.1", upstream: "http://127.0.0.1:3010/auth", host: "localhost:8080"
2020/07/17 07:13:39 [error] 27#27: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET /auth HTTP/1.1", upstream: "http://127.0.0.1:3010/auth", host: "localhost:8080"
